README.md

```diff
@@ -12,6 +12,8 @@ place.
 Please check the [dedicated documentation][2] to see how you can install and
 run DocSearch yourself.
 
+This project supports Python 2.7 and Python 3.4+
```

Comment:
To [continue the requirements of scrapy](https://github.com/scrapy/scrapy#requirements).
I want to avoid any big breaking change yet, might change it in the future.

Reply:
It matters for people who forked the repo and use python 2.7. We might drop this support at some point